Assignment5.txt

```diff
@@ -0,0 +1,23 @@
+Ashank Kumar
+
+1a. Passing by value means a copy is made of the passed value to be used locally by the function.
+
+int fnx(int a)
+{
+	a = 0;
+	return a;
+}
+
+1b. Passing by reference means the memory address of the passed variable is sent to the function, allowing the function to change the variables value globally, not just locally.
```

Comment:
right - see http://stackoverflow.com/questions/12953496/what-happens-during-pass-by-reference-in-c